sumo/cli/bandplot.py

```diff
@@ -165,6 +165,8 @@ def bandplot(filenames=None, code='vasp', prefix=None, directory=None,
 
             If ``atoms`` is not set or set to ``None`` then all atomic sites
             for all elements will be considered.
+        spin (:obj:`str`, optional): Plot a spin-polarised band structure,
```

Comment:
Could this be  *"Plot only one channel of a spin-polarised ..."* to make it clearer that `None` will plot both channels?

Reply:
Thanks for the helpful feedback! I've changed this for bandplot and dosplot as shown below:
![image](https://user-images.githubusercontent.com/51478689/69966003-674bcb00-150d-11ea-927f-030d391d5ac3.png)